custom_button/CustomButton.py

```diff
@@ -0,0 +1,101 @@
+import tkinter as tk
+from tkinter import *
+from external.constants import FONT_FAMILY, FONT_SIZE
+
+class CustomButton:
+    """Class, whick represents custom button."""
+
+    color = "#21A2F1"
+    activebackground = "white"
+    width = 70
+    height = 40
+    index = 0
+    """Static arguments which represent button
+
+    Params:
+        color (str): background colour of button
+        activebackground (str): colour of activebackground
+        width (int): the value of width of button
+        height (int): the value of height of button
+        index (int): the index of the button"""
+
+    def __init__(self, master, your_text, button_command):
+        """Construct new CustomButton object
+
+        Params:
+            master (tkinter): base for your app
```

Comment:
I suggest to change documentation into: `parent of your app`. AFAIK it's usually called parent, but variable name `master` can stay